### PR TITLE
Add MDEX support (BSC) [TKR-426]

### PR DESCRIPTION
**Pull request by eobbad**
## Description

Added MDEX on BSC

## Testing instructions

Simbot BSC

## Types of changes

New feature (non-breaking change which adds functionality) 

## Checklist:

-   [x] Prefix PR title with `[WIP]` if necessary.
-   [x] Add tests to cover changes as needed.
-   [x] Update documentation as needed.
-   [x] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

**Review by idokleinman — DISMISSED** [review]
I think that the files under /curve_automation don't belong in this PR?

**Comment by eobbad** [comment]
> 

yeah, going to remove these, just wanted to make the PR so I could get help with my error i'm getting.

File: packages/asset-swapper/src/utils/market_operation_utils/bridge_source_utils.ts [hunk 1]
```diff
@@ -545,7 +546,9 @@ export function uniswapV2LikeRouterAddress(
         | ERC20BridgeSource.SpiritSwap
         | ERC20BridgeSource.BiSwap
         | ERC20BridgeSource.Yoshi
-        | ERC20BridgeSource.MeshSwap,
+        | ERC20BridgeSource.MeshSwap
```
Comment (kh-chang) [inline packages/asset-swapper/src/utils/market_operation_utils/bridge_source_utils.ts:549]: missing comma after MeshSwap

File: packages/asset-swapper/src/utils/market_operation_utils/bridge_source_utils.ts [hunk 2]
```diff
@@ -545,7 +546,9 @@ export function uniswapV2LikeRouterAddress(
...
+          ERC20BridgeSource.MDEX,
```
Comment (kh-chang) [inline packages/asset-swapper/src/utils/market_operation_utils/bridge_source_utils.ts:550]: missing `|`

**Review by eobbad — COMMENTED** [review]
done

**Pull request merged**